README.md

```diff
@@ -40,6 +42,7 @@ The goals of the wiki are explained in the [Manifest](MANIFEST.md) and contribut
 - [Exceptions](General/Exceptions.md)
 - [Sorting collections](General/SortingCollections.md)
 - [Refactoring](General/Refactorings.md) - Documentation on the usage of Pharo refactoring's tool
+- [Migrating Streams from Pharo 6.1 to Pharo 7.0](Migration/MigrationToPharo7.md)
```

Comment:
I liked to have a separated section for migration guides. I think it makes it easier to find info when you are looking for resources on migration or just general documentation on Pharo. 

Do you have an argument against it?

Reply:
My argument _for_ putting the word streams is that I am always looking to update those in old examples, etc. So, I go back to the wiki, search for Streams on the README and there is nothing :( I always have to remember that it's buried under Migration. Then, on the migration page, there is only stuff about streams. 

I agree with the idea of having a general migration page. But for now there is only Streams. This is reminiscent of the "speculative generality" code smell.

Reply:
I agree to add "Stream" to the name. 

I was talking about sections here.